stdlib/2and3/binascii.pyi

```diff
@@ -0,0 +1,34 @@
+# Stubs for binascii
+
+# Based on http://docs.python.org/3.2/library/binascii.html
+
+import sys
+from typing import Union, Text
+
+if sys.version_info < (3,):
+    # Python 2 accepts unicode ascii pretty much everywhere. 
+    _Bytes = Union[bytes, Text]
+else:
+    # Python 3 only accepts bytes on the b2a methods.
+    _Bytes = bytes
+
+def a2b_uu(string: Union[Text, bytes]) -> bytes: ...
+def b2a_uu(data: _Bytes) -> bytes: ...
+def a2b_base64(string: Union[Text, bytes]) -> bytes: ...
+def b2a_base64(data: _Bytes) -> bytes: ...
+def a2b_qp(string: Union[Text, bytes], header: bool = ...) -> bytes: ...
+def b2a_qp(data: _Bytes, quotetabs: bool = ..., istext: bool = ...,
+             header: bool = ...) -> bytes: ...
+def a2b_hqx(string: Union[Text, bytes]) -> bytes: ...
+def rledecode_hqx(data: _Bytes) -> bytes: ...
+def rlecode_hqx(data: _Bytes) -> bytes: ...
+def b2a_hqx(data: _Bytes) -> bytes: ...
+def crc_hqx(data: _Bytes, crc: int) -> int: ...
```

Comment:
It's positional only in both.  Technically this entire module is positonal only everywhere.